compose/src/main/java/dev/marcellogalhardo/retained/compose/ComposeRetained.kt

```diff
@@ -59,19 +59,16 @@ public inline fun <reified T : Any> retain(
 @ExperimentalRetainedApi
 @Composable
 public inline fun <reified T : Any> retainInActivity(
-    owner: ViewModelStoreOwner?,
     key: String = T::class.java.name,
     noinline instantiate: @DisallowComposableCalls (RetainedEntry) -> T,
-): T = retain(
-    key = key,
-    owner = if (owner != null) {
-        owner
-    } else {
-        val context = LocalContext.current
-        remember { context.findActivity() }
-    },
-    instantiate = instantiate,
-)
+): T {
+    val context = LocalContext.current
+    return retain(
+        key = key,
+        owner = remember { context.findActivity() },
+        instantiate = instantiate,
+    )
+}
```

Comment:
I couldn’t think of any non-awkward way of keeping the function in the expression body style like the rest. But happy to tweak it if you have any ideas.